README.md

```diff
@@ -1,76 +1,61 @@
-examples for the many LSP backends out there, and (2) a reference for the
-current best practice (or most popular) regarding choice of server, setup, etc.
-
-## Requires Nvim 0.5 HEAD
-
-While Nvim LSP undergoes development, the configs in this repo assume that you
-are using the latest [Nvim HEAD/nightly build](https://github.com/neovim/neovim/releases/tag/nightly).
-
-Update Nvim and nvim-lspconfig before reporting an issue.
+This repo is :
 
-## Contributions
-
-It's up to you to send improvements so that these configs align with current
-best practices for a given language.
-
-1. Read [CONTRIBUTING.md](CONTRIBUTING.md).
-   Ask questions in [Neovim Gitter](https://gitter.im/neovim/neovim).
-2. Choose a language from [the coc.nvim wiki](https://github.com/neoclide/coc.nvim/wiki/Language-servers) or
-  [emacs-lsp](https://github.com/emacs-lsp/lsp-mode#supported-languages).
-3. Create a new file at `lua/lspconfig/SERVER_NAME.lua`.
-   - Copy an [existing config](https://github.com/neovim/nvim-lspconfig/blob/master/lua/lspconfig/)
-     to get started. Most configs are simple. For an extensive example see
-     [texlab.lua](https://github.com/neovim/nvim-lspconfig/blob/master/lua/lspconfig/texlab.lua).
+* a place for users of Neovim's language server client to collaboratively provide starting 
+  examples for the many LSP backends out there
+* a reference for the current best practice (or most popular) regarding 
+  choice of server, setup, etc.
 
 ## Install
 
-- Requires [Nvim HEAD/nightly](https://github.com/neovim/neovim/releases/tag/nightly) (v0.5 prerelease).
-- nvim-lspconfig is just a plugin. Install it like any other Vim plugin, e.g. with [vim-plug](https://github.com/junegunn/vim-plug):
+* Requires [Neovim HEAD/nightly](https://github.com/neovim/neovim/releases/tag/nightly) (v0.5 prerelease). The configs in this repo
+  assume that you are using the latest [Neovim HEAD/nightly build](https://github.com/neovim/neovim/releases/tag/nightly). 
+  Update Neovim and nvim-lspconfig before reporting an issue.
+
+* nvim-lspconfig is just a plugin. Install it like any other Vim plugin, e.g. with [vim-plug](https://github.com/junegunn/vim-plug):
   ```
   :Plug 'neovim/nvim-lspconfig'
   ```
-- Call `:packadd nvim-lspconfig` in your vimrc if you installed nvim-lspconfig to `'packpath'` or if you use a package manager such as minpac.
-
-## Usage
-
-Each config provides a `setup()` function, to initialize the server with
-reasonable defaults and some server-specific things like commands or different
-diagnostics.
-
-```lua
-vim.cmd('packadd nvim-lspconfig')  -- If installed as a Vim "package".
-require'lspconfig'.<config>.setup{name=…, settings = {…}, …}
-```
 
-If you want to add this to your vimrc, you will need to enclose it in a `lua` block.
+Note: if you installed nvim-lspconfig to `'packpath'` you need to call 
+`:packadd nvim-lspconfig` in your vimrc.  This is the case for only 
```

Comment:
This is actually no longer true (unless you put it in `pack/*/opt`, where this is intended).

Reply:
I've never manually managed pack path. Are we safe to just delete this line?

Reply:
I think there are three possibilities here:
1. People use vim-plug or a similar plugin manager that works by managing rtp.
2. People use min-pac or packer that works with vim native packages.
3. People install plugins/packages by hand.

In case 1 and 2, the respective plugin/package manager should have ample documentation for adding plugins/packages. In case 3, the user knows what they are doing (hopefully) :)

The only issue would be if lspconfig did not behave like a normal plugin/package (which used to be the case for packages until a short while ago because Lua handling in packages had hiccups, but this has been fixed).

So, yes, I'd say it's safe to delete the note (especially since you -- sensibly -- give an example for vim-plug).

Reply:
Agreed. It's unlikely this is someone's *first* plugin, and I had 2 of the 4 non-vim users I had test out the readme get confused by this.